modules/gapi/src/backends/cpu/gcpuvideo.cpp

```diff
@@ -80,12 +80,51 @@ GAPI_OCV_KERNEL(GCPUCalcOptFlowLKForPyr, cv::gapi::video::GCalcOptFlowLKForPyr)
     }
 };
 
+/*namespace cv
+{
```

Comment:
Dead code?

It is better to wrap huge blocks of code through `#if 0` instead of C++ comments.

Also please left comment that is it commented? `TODO` for GPU support or something just doesn't work.

Reply:
Removed